src/labkey/Domain.ts

```diff
@@ -157,6 +157,33 @@ export function get(config: GetDomainOptions): void {
 
 }
 
+/**
+ * Gets a domain design.
+ */
+export function getDomainDetails(config: GetDomainOptions): void {
```

Comment:
Good idea on only exposing the new and preferred version.

Reply:
From what I can see, there is only one usage of Domain.get() out there in our various usage of this package, in labkey-ui-components domainproperties/actions.ts fetchDomain(). I think it would be pretty easy to convert this usage over to Domain.getDomainDetails(). I give that a try tomorrow morning.

Reply:
It was a very simply change in labkey-ui-components to use Domain.getDomainDetails instead of Domain.get, so I opted to remove Domain.get from here all together. Let me know what you guys think. Note: I also merged in master for labkey-api-js.

Reply:
Works for me!

Reply:
You removed `Domain.get()`, however, we still need to be compatible with core/Domain.js until v1.0 as this code will replace the implementation and `Domain.get()` will need to be available.

I think it would be good to still alias `.get()` to `.getDomainDetails()`. Something like:

```ts
/**
 * Gets a domain design.
 */
export function getDomainDetails(config: GetDomainOptions): void { /* impl */ }

/**
 * Gets a domain design. This is a deprecated alias for [[getDomainDetails]].
 * @deprecated
 */
export const get = getDomainDetails;
```

Reply:
Added back. thanks for the code suggestion.

Reply:
Looks good. Thanks for addressing this.